### ISSUE TITLE
Only works with new files

### ISSUE THREAD

**Issue by surfer190** [borisguery/git-keywords-checker#7]
I have noticed that this only works with new files.

@borisguery @hkdobrev 

Modifying an existing file with the debug commands does not detect and shows `All files are clean`

**Comment by hkdobrev** [comment]
@surfer190 Could you please modify an existing file, stage it and see if this command gives you any output:

```
git diff-index --cached --name-only --diff-filter=ACMR HEAD
```

The `M` in the `--diff-filter` option indicates files which were modified.
